Content.Shared/Blocking/BlockingSystem.cs

```diff
@@ -70,6 +75,29 @@ private void OnToggleAction(EntityUid uid, BlockingComponent component, ToggleAc
         if(args.Handled)
             return;
 
+        if (component.User != null)
+        {
+            foreach (var shield in _handsSystem.EnumerateHeld(component.User.Value))
+            {
+                if (TryComp<BlockingComponent>(shield, out var otherBlockComp))
+                {
+                    if ((!component.IsBlocking && otherBlockComp.IsBlocking))
+                    {
+                        StopBlocking(shield, otherBlockComp, args.Performer);
+                        StartBlocking(uid, component, args.Performer);
+                        return;
+                    }
+
+                    if (component.IsBlocking && !otherBlockComp.IsBlocking)
+                    {
+                        StopBlocking(uid, component, args.Performer);
+                        StartBlocking(shield, otherBlockComp, args.Performer);
+                        return;
+                    }
+                }
+            }
```

Comment:
should the code even support being able to block with multiple items in multiple hands? i feel like it just makes stuff way jankier. would be cleaner if you just didn't allow it if they're already blocking or something

Reply:
Anything I tried did not solve this. It would end up where it would only stop you from blocking/unblocking with your  main shield and then crash when another block toggle was used. So this was the only solution that worked and I honestly have no clue otherwise on how to fix this.

Reply:
@ShadowCommander really helped me out on this one so thanks to them.